starknet_py/tests/e2e/fixtures/constants.py

```diff
@@ -1,19 +1,33 @@
 import os
 from pathlib import Path
 
+# -------------------------------- TESTNET -------------------------------------
+
 TESTNET_ACCOUNT_PRIVATE_KEY = (
     "0x61910356c5adf66efb65ec3df5d07a6e5e6e7c8b59f15a13eda7a34c8d1ecc4"
 )
 TESTNET_ACCOUNT_ADDRESS = (
     "0x59083382aadec25d7616a7f48942d72d469b0ac581f2e935ec26b68f66bd600"
 )
 
+# -------------------------------- INTEGRATION ---------------------------------
+
 INTEGRATION_ACCOUNT_PRIVATE_KEY = "0x1234"
 
 INTEGRATION_ACCOUNT_ADDRESS = (
     "0x4321647559947e9109acecb329e57594bcc3981a6118bbbfeaa9f698874bcd5"
 )
 
+INTEGRATION_NODE_URL = "http://188.34.188.184:9545/rpc/v0.4"
```

Comment:
Not sure about adding this to the lib 👀 
Maybe we could make it work by passing some argument when running tests? Hmm